firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+TabScrollHandlerDelegate.swift

```diff
@@ -20,13 +20,26 @@ extension BrowserViewController: TabScrollHandler.Delegate {
         return baseOffset + UX.minimalHeaderOffset
     }
 
+    /// Interactive toolbar transition.
+    /// Top bar moves in [headerOffset, 0] using `originTop - clampProgress`
+    /// Bottom bar moves in [0, height]using `originBottom + clampProgress`
```

Comment:
nit; one space missing
```suggestion
    /// Bottom bar moves in [0, height] using `originBottom + clampProgress`
```